coops-ui/src/views/StandaloneDirectorsFiling.vue

```diff
@@ -553,7 +553,7 @@ export default {
         }
       }
       if (setting === 'add' && !added) {
-        this.filingData.push({ filingTypeCode: filing, entityType: 'CP' })
+        this.filingData.push({ filingTypeCode: filing, entityType: this.entityType })
```

Comment:
Doesn't this fix also apply to the Annual Report page (directors section)?

Reply:
Certainly, i imagine it would. I didn't want to stray outside the scope of the ticket but it makes sense filings director changes from the annual report should be included. Is there any other locations this filing code would need to be changed? Will this need to be applied to address-change filings, AR filings etc as well?

Reply:
* Updated and ready for review 👍